examples/ui-demo/src/utils/config.ts

```diff
@@ -0,0 +1,240 @@
+import { arbitrumSepolia } from "@account-kit/infra";
+export const entryPointAddress = "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
```

Comment:
where is this used?

Reply:
Artifact from experimenting with the gassless minter pattern, will remove